gmso/utils/expression.py

```diff
@@ -0,0 +1,293 @@
+import warnings
+
+import sympy
+
+import unyt as u
+
+from gmso.utils.misc import unyt_to_hashable
+
+class _PotentialExpression:
+    """A general Expression class with parameters
+
+    Parameters
```

Comment:
Maybe add a note here about the underscore class and its intended usage. Will be helpful for developers.